docs/database-engine/availability-groups/windows/replicate-track-change-data-capture-always-on-availability.md

```diff
@@ -167,7 +167,7 @@ helpviewer_keywords:
      In general, you should use domain logins for client access to change data residing in databases that are members of Always On availability groups. To ensure continued access to change data after failover, the domain user will need access privileges on all of the hosts supporting availability group replicas. If a database user is added to a database in a primary replica, and the user is associated with a domain login, the database user is propagated to secondary databases and continues to be associated with the specified domain login. If the new database user is associated with a SQL Server authentication login, the user at the secondary databases will be propagated without a login. While the associated [!INCLUDE[ssNoVersion](../../../includes/ssnoversion-md.md)] authentication login could be used to access change data at the primary where the database user was originally defined, that node is the only one where access would be possible. The [!INCLUDE[ssNoVersion](../../../includes/ssnoversion-md.md)] authentication login would not be able to access data from any secondary database, nor from any new primary databases other than the original database where the database user was defined.  
      
 -   **Disabling Change Data Capture**  
-If Change Data Capture needs to be disabled on a database that is part of an Always On Availability Group, then you will need to perform more steps to ensure that log truncation is not affected. You will need to implement one of the following steps to prevent Change Data Capture from blocking log truncation after disabling Change Data Capture:
+If Change Data Capture needs to be disabled on a database that is part of an Always On Availability Group and your SQL version is below [SQL 2016 SP2](https://support.microsoft.com/en-us/topic/kb4092045-fix-lsn-truncation-occurs-in-alwayson-ag-when-disabling-the-change-data-capture-feature-in-sql-server-2016-cf5f641f-5550-caa0-8bb9-418294101bf3) , then you will need to perform more steps to ensure that log truncation is not affected. You will need to implement one of the following steps to prevent Change Data Capture from blocking log truncation after disabling Change Data Capture:
```

Comment:
```suggestion
If you need to disable Change Data Capture on a database that is part of an Always On availability group and your SQL Server version is below [SQL Server 2016 SP2](https://support.microsoft.com/en-us/topic/kb4092045-fix-lsn-truncation-occurs-in-alwayson-ag-when-disabling-the-change-data-capture-feature-in-sql-server-2016-cf5f641f-5550-caa0-8bb9-418294101bf3), then you will need to implement one of the following steps to prevent Change Data Capture from blocking log truncation after Change Data Capture is disabled:
```